openbas-front/src/admin/components/simulations/simulation/injects/ExerciseInjects.tsx

```diff
@@ -70,6 +74,10 @@ const ExerciseInjects: FunctionComponent = () => {
     dispatch(fetchExerciseArticles(exerciseId));
     dispatch(fetchVariablesForExercise(exerciseId));
     dispatch(fetchExerciseInjectExpectations(exerciseId));
+    dispatch(fetchExerciseEndpoints(exerciseId));
+    dispatch(fetchExerciseAssetGroups(exerciseId));
+    dispatch(fetchExerciseDocuments(exerciseId));
+    dispatch(fetchExerciseChannels(exerciseId));
```

Comment:
Why adding this here ?

Reply:
As we are fetching documents, endpoints, asset groups and channel per scenario/simulation, i put it at this level instead of doing it at the inject scenario. This way, we make one call when loading all injects instead of for every inject. Where would you prefer I put it?